yesod-test/ChangeLog.md

```diff
@@ -1,5 +1,8 @@
 # ChangeLog for yesod-test
 
+## 1.6.16
+
+* Fix yesod-websockets chat examples ([#1790](https://github.com/yesodweb/yesod/pull/1790)).
```

Comment:
I think this should appear in the yesod-websockets changelog, not the yesod-test changelog.